titan/params/prep.yml

```diff
@@ -13,6 +13,10 @@ prep:
     type: float
     min: 0
     max: 1
+  as_prob:
```

Comment:
maybe `target_as_prob` - how does target differ from the `coverage` we have in demographics?